app/routines/update_user_salesforce_info.rb

```diff
@@ -203,10 +209,20 @@ def cache_contact_data_in_user!(contact, user)
         :pending_faculty
       when /Rejected/
         :rejected_faculty
-      when nil
+      when NilClass
```

Comment:
I should know why this is important but do not - fill me in?

Reply:
It's not important, because in ruby `nil == nil` but I feel bad about it because that's not true in SQL for example.